js/app.js

```diff
@@ -1,4 +1,48 @@
 /* setup your angular app here */
+var app = angular.module('F&V', []);
+
+app.controller('mainCtrl', ['$scope', function($scope) {
+  $scope.fruits = []
+  $scope.veggies = []
+  $scope.foods = fruits.concat(vegetables);
+
+  $scope.moveLeft = function(idx) {
+    console.log('You have opted to move this item one div to the left');
+    console.log(idx);
+    $scope.fruits.push($scope.foods[idx]);
+    $scope.foods.splice(idx, 1);
+  }
+
+  $scope.moveRight = function(idx) {
+    console.log('You have opted to move this item oine div to the right');
+    console.log(idx);
+    $scope.veggies.push($scope.foods[idx]);
+    $scope.foods.splice(idx, 1);
+  }
+
+  $scope.moveBackLeft = function(idx) {
+    console.log('You have opted to move this item one div to the left');
+    console.log(idx);
+    $scope.foods.push($scope.veggies[idx]);
+    $scope.veggies.splice(idx, 1);
+  }
+
+  $scope.moveBackRight = function(idx) {
+    console.log('You have opted to move this item oine div to the right');
+    console.log(idx);
+    $scope.foods.push($scope.fruits[idx]);
+    $scope.fruits.splice(idx, 1);
+    console.log(fruits);
+    console.log(vegetables);
+    console.log($scope.fruits.sort);
+
+  // }
+
+  // $scope.checkWin = function(idx) {
+  //   console.log($scope.fruits.sort);
+
```

Comment:
here is my win check function:
    
    $scope.winCheck = function(){
        
        for (var i = 0; i < $scope.allFruit.length; i++) {
            if ($scope.originalFruit.indexOf($scope.allFruit[i]) == -1) {
                $scope.fail = true;
                return false;
            }
        }
        for (var i = 0; i < $scope.allVegetables.length; i++) {
            if ($scope.originalVegetables.indexOf($scope.allVegetables[i]) == -1) {
                $scope.fail = true;
                return false;
            }
        }
        $scope.win = true;
        console.log($scope.win);
    }